src/resty/aws/credentials/CredentialProviderChain.lua

```diff
@@ -36,6 +36,7 @@ CredentialProviderChain.defaultProviders = {} do
   add_if_exists("RemoteCredentials") -- since "ECSCredentials" doesn't exist? and for ECS RemoteCredentials is used???
   add_if_exists("ProcessCredentials")
   add_if_exists("TokenFileWebIdentityCredentials")
+  add_if_exists("ProfileCredentials")
```

Comment:
It would be better that we also follow the other official SDK standard and let this credential provider take precedence in the default provider chain

See
https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials
https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-credentials-node.html
https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html